### PR TITLE
DOCS-11165 ACM 2.0

### PR DESCRIPTION
**Pull request by wwarner-salesforce**

### CODE REVIEW

**Review by ferferrarazzo — DISMISSED** [review]

File: modules/ROOT/pages/acm-lightning-components.adoc [hunk 1]
```diff
@@ -391,6 +391,9 @@ The component shows the application description, application URL, redirect URIs,
 You can specify values for the following parameters of the Application Details:
 
 Application:: Specifies the API to show.
+Enable Application Editing:: Enables editing application details such as the name and description.
++
+When application editing is enabled, the component displays the *Reset Credentials* button. By default this button works on an application with the client provider type Anypoint, and displays an error on an application with a different client provider type such as OpenID. An administrator can customize the button to reset credentials successfully on an application with a client provider type other than Anypoint.
```
Comment (kbostrom) [inline modules/ROOT/pages/acm-lightning-components.adoc]:
```suggestion
When application editing is enabled, the component displays the *Reset Credentials* button. By default, this button works on an application with the client provider type Anypoint, and displays an error on an application with a different client provider type such as OpenID. An administrator can customize the button to reset credentials successfully on an application with a client provider type other than Anypoint.
```
Are these the regular narrative names of client provider types, or are these referring to syntax representations? If the latter, use mono.

File: modules/ROOT/pages/update-acm.adoc [hunk 2]
```diff
@@ -1,13 +1,21 @@
...
 include::_attributes.adoc[]
 endif::[]
 :imagesdir: ../assets/images/
 
-Keep your Anypoint API Community Manager instance up to date to take advantage of new and improved functionality in your community.
+Upgrades are applied automatically to Anypoint API Community Manager versions 2.0 and later.
+
+Upgrades with new features are applied to sandbox environments four weeks before they are applied to production environments. Upgrades with only bug fixes are applied to sandbox environments two weeks before they are applied to production environments.
+
+To ensure the community administrator controls how their audience receives community changes, new functionality in upgrades is typically disabled until the community administrator enables it.
```
Comment (kbostrom) [inline modules/ROOT/pages/update-acm.adoc]: ensure that, always, per style ref:
```suggestion
To ensure that the community administrator controls how their audience receives community changes, new functionality in upgrades is typically disabled until the community administrator enables it.
```

File: modules/ROOT/pages/update-acm.adoc [hunk 3]
```diff
@@ -303,6 +312,37 @@ Manual configuration:
 .. Click *Save*.
 . Verify that your permissions match those in the xref:create-community.adoc#configure-profiles-and-permissions[Configure Profiles and Permissions] documentation.
 
+Return to the main list of upgrade instructions to continue.
+
+[[post-20]]
+== Postinstallation Steps When Upgrading from a Version Earlier Than 2.0
+
+If you are upgrading from a package version earlier than 2.0, use either the automatic or manual configuration methods to update the permissions of the member and guest profiles. Both profiles require field-level read access to the *AssetId* field in the *AnypointApiInstances* object.
```
Comment (kbostrom) [inline modules/ROOT/pages/update-acm.adoc]:
```suggestion
If you are upgrading from a package version earlier than 2.0, use either the automatic or manual configuration method to update the permissions of the member and guest profiles. Both profiles require field-level read access to the *AssetId* field in the *AnypointApiInstances* object.
```

File: modules/ROOT/pages/update-acm.adoc [hunk 4]
```diff
@@ -303,6 +312,37 @@ Manual configuration:
...
+
+Automatic configuration:
+
+. From the Anypoint API Community Manager control panel, open the Salesforce App Launcher and search for `API community manager diagnostics`.
+. In the *User Access* tab, in the section *Guest User*, click *Fix all*.
+. In the *User Access* tab, in the section *Member User*, click *Fix all*.
```
Comment (kbostrom) [inline modules/ROOT/pages/update-acm.adoc]: We could probably combine these:
```suggestion
. In the *User Access* tab, in the sections *Guest User* and *Member User*, click *Fix all*.
```

File: modules/ROOT/pages/update-acm.adoc [hunk 5]
```diff
@@ -303,6 +312,37 @@ Manual configuration:
...
+.. Ensure that the *AssetId* field has *Read* access.
+.. Save.
+. Configure the guest profile permissions:
+.. Open the community builder.
+.. Click *Settings* > *General*.
+.. Open the *Guest* profile.
+.. Click *Object settings* > *AnypointApiInstances*.
+.. Ensure that the *AssetId* field has *Read* access.
+.. Save.
+. Verify that your permissions match those in the xref:create-community.adoc#configure-profiles-and-permissions[Configure Profiles and Permissions] documentation.
```
Comment (kbostrom) [inline modules/ROOT/pages/update-acm.adoc:343]: ditto re. verification of instructions

File: modules/ROOT/pages/update-acm.adoc [hunk 6]
```diff
@@ -303,6 +312,37 @@ Manual configuration:
...
+== Postinstallation Steps When Upgrading from a Version Earlier Than 2.0
+
+If you are upgrading from a package version earlier than 2.0, use either the automatic or manual configuration methods to update the permissions of the member and guest profiles. Both profiles require field-level read access to the *AssetId* field in the *AnypointApiInstances* object.
+
+Automatic configuration:
+
+. From the Anypoint API Community Manager control panel, open the Salesforce App Launcher and search for `API community manager diagnostics`.
+. In the *User Access* tab, in the section *Guest User*, click *Fix all*.
+. In the *User Access* tab, in the section *Member User*, click *Fix all*.
+. Verify that your permissions match those in the xref:create-community.adoc#configure-profiles-and-permissions[Configure Profiles and Permissions] documentation.
```
Comment (kbostrom) [inline modules/ROOT/pages/update-acm.adoc:326]: Wait...you go through this whole task and then doublecheck it against some other doc?

**Review by kbostrom — DISMISSED** [review]
CE

File: modules/ROOT/pages/acm-lightning-components.adoc (hunk 1 above)
Comment (wwarner-salesforce) [reply modules/ROOT/pages/acm-lightning-components.adoc]: Fixed the comma. These are regular narrative names.

File: modules/ROOT/pages/update-acm.adoc (hunk 2 above)
Comment (wwarner-salesforce) [reply modules/ROOT/pages/update-acm.adoc]: Thank you! I will try to remember this. Fixed.

File: modules/ROOT/pages/update-acm.adoc (hunk 3 above)
Comment (wwarner-salesforce) [reply modules/ROOT/pages/update-acm.adoc]: Fixed.

File: modules/ROOT/pages/update-acm.adoc (hunk 4 above)
Comment (wwarner-salesforce) [reply modules/ROOT/pages/update-acm.adoc]: Yes, that's better. Fixed.

File: modules/ROOT/pages/update-acm.adoc (hunk 5 above)
Comment (wwarner-salesforce) [reply modules/ROOT/pages/update-acm.adoc:343]: see below

File: modules/ROOT/pages/update-acm.adoc (hunk 6 above)
Comment (wwarner-salesforce) [reply modules/ROOT/pages/update-acm.adoc:326]: Yes. Customers have said the upgrade configurations are a pain point, and the ACM team is working on a comprehensive solution to automate all of this. The ACM team discussed whether to remove the comparison to the permissions on the community page, which increases the risk of errors, troubleshooting, or support cases, or keep the comparison, which makes the instructions more complex, and decided to keep it until the more comprehensive solution is released.

**Review by kbostrom — APPROVED** [review]
RA

**Pull request merged**